com.vrcfury.vrcfury/Editor/VF/Utils/AnimatorControllerExtensions.cs

```diff
@@ -28,6 +29,18 @@ public static class AnimatorControllerExtensions {
         }
 
         public static void RewriteParameters(this AnimatorController c, Func<string, string> rewriteParamName, bool includeWrites = true) {
+            IImmutableSet<StateMachineBehaviour> behaviors = null;
+
+            // Collect data before calling the 'rewriteParamName' callback, which could add 
```

Comment:
What can cause this to happen?

Reply:
Not sure if it's still the same after your changes but previously when adding the smoothing in the callback, the parameters of the smoothing layers would also get rewritten

Reply:
AH hmm, that is tricky. It's really cleaner for the smoothing to happen before RewriteParameters is called (so we can just do one pass over the param list and smooth the ones that need smoothed), but you're right that then it runs into this issue.